src/components/MovieCard.js

```diff
@@ -0,0 +1,14 @@
+import { MovieRating } from "./MovieRating";
+
+export function MovieCard({ movie }) {
+  return (
+    <div className="movie-card">
+      <h2 className="movie-title">{movie.title}</h2>
+      <p>{movie.description}</p>
+
+      <MovieRating movie={movie} />
```

Comment:
This component only handles the rating, so no need to pass the complete `movie` object, only the `rating` would suffice.